data/faq.html

```diff
@@ -259,10 +288,22 @@ <h3 lang="es">Datos de los mapas</h3>
 el dispositivo. Ten en cuenta que incluso los mapas más recientes se marcarán como obsoletos
 si los descargas utilizando un PC.</dd>
 
+<dd lang="fr">Votre téléchargement peut échouer suite à un problème de réseau temporaire ou de paramêtre de réseau spécifique du fournisseur/routeur.
+. Merci de réessayer plus tard ou d'utiliser un point d'accès wifi différent. En complément, assurez vous d'avoir suffisamment d'espace disponible pour télécharger les cartes.
+
+<p></p>
+Si vous avez une connexion internet lente, vous pouvez <a href="http://direct.mapswithme.com/direct/latest/" class="link">télécharger
```

Comment:
Please make the lines short enough (around 110 symbols) so that we can see them in the GitHub diff.